python/ray/tune/pbt.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import numpy as np
+import random
+import math
+import copy
+
+from ray.tune.trial_scheduler import FIFOScheduler, TrialScheduler
+from ray.tune.trial import Trial
+import ray
```

Comment:
Note: two newlines between declarations. You can run the linter with `flake8 python/ray/tune/**/*.py test/trial_*.py` in the Ray dir.